tools/python/mwm/dump_mwm.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import sys, os.path, random
+import json
+from mwm import MWM
+
+if len(sys.argv) < 2:
+    print('Dumps some MWM structures.')
+    print('Usage: {0} <country.mwm>'.format(sys.argv[0]))
+    sys.exit(1)
+
+mwm = MWM(open(sys.argv[1], 'rb'))
+mwm.read_types(os.path.join(os.path.dirname(sys.argv[0]), '..', '..', '..', 'data', 'types.txt'))
+print('Tags:')
+tvv = sorted([(k, v[0], v[1]) for k, v in mwm.tags.items()], key=lambda x: x[1])
+for tv in tvv:
+    print('  {0:<8}: offs {1:9} len {2:8}'.format(tv[0], tv[1], tv[2]))
+v = mwm.read_version()
+print('Format: {0}, version: {1}'.format(v['fmt'], v['version'].strftime('%Y-%m-%d %H:%M')))
+print('Header: {0}'.format(mwm.read_header()))
+print('Metadata count: {0}'.format(len(mwm.read_metadata())))
+
+cross = mwm.read_crossmwm()
+if cross:
+    print('Outgoing points: {0}, incoming: {1}'.format(len(cross['out']), len(cross['in'])))
+    print('Outgoing regions: {0}'.format(set(cross['neighbours'])))
+
+print('Sample features:')
+# Print 5 random features ~500 features apart
```

Comment:
This is actually irrelevant. You are right by saying features will be ~10k features apart, I've fixed the comment. The point is to get any random 5 features, not random 5 of the whole file or anything. I've implemented the fastest and simplest algorithm for this.